platform/linux-generic/pktio/dpdk.c

```diff
@@ -1900,6 +1901,14 @@ static int dpdk_open(odp_pktio_t id ODP_UNUSED,
 		return -1;
 	}
 
+	memset(&eth_conf, 0, sizeof(eth_conf));
+	ret = rte_eth_dev_configure(pkt_dpdk->port_id, 1, 1, &eth_conf);
+	if (ret) {
+		_ODP_ERR("Failed to configure device: err=%d, port=%" PRIu8 "\n",
+			 ret, pkt_dpdk->port_id);
+		return -1;
+	}
+
```

Comment:
Now the `rte_eth_dev_set_mtu()` comment and check in `dpdk_init_capability()` could be updated.